Lide 001

hark130 opened the pull request:
REVIEW INSTRUCTIONS:

- Follow the `Start Here` steps on the README
- Check the README for accuracy, detail, readability, etc

If it works, merge it.
If not, I'll need some feedback.  Screenshots and copy/paste Vagrant output would be appreciated.

[LIDE-001](https://github.com/globalinfotek/linux-dev-practice/issues/1)

hark130 committed:
LIDE-001: Vagrant file initial commit

vagrant up

Changes to be committed:
	new file:   Vagrantfile

hark130 committed:
LIDE-001: Vagrant provision script

Extensible configuration changes executed by Vagrant at "creation" time.
Trying to keep the Dockerfile (mostly) free of chaos

Changes to be committed:
	new file:   devops/vagrant/provision.sh

hark130 committed:
LIDE-001: Updated README w/ descr, usage, and POCs

Changes to be committed:
	modified:   README.md

hark130 committed:
LIDE-001: Updated Vagrantfile

Removed hard-coded values for CPUs and Memory
My CDE was unusable
Best to let the user find the right values for their system
Notes added the Usage section of the README

Changes to be committed:
	modified:   Vagrantfile

hark130 committed:
LIDE-001: Added Vagrant files to gitignore

Changes to be committed:
	modified:   .gitignore

hark130 committed:
LIDE-001: Polished README

Made the POCs pretty

suirad committed:
fix vagrant file; update readme for vm-in-vm

suirad commented:
There was a small issue with a line in the vagrant file. Idk where that line comes from or how it works for your setup but I checked the vagrant docs and changed the `auto_update` line myself. Error persisted between apt installed vagrant and latest vagrant installed from the site.

I also added notes to the readme for those wanting to run vagrant inside of a vmware-vm. My setup is similar.

suirad commented:
@hark130 I am getting an access error trying to push to the repo.

hark130 commented:
> There was a small issue with a line in the vagrant file. Idk where that line comes from or how it works for your setup but I checked the vagrant docs and changed the `auto_update` line myself. Error persisted between apt installed vagrant and latest vagrant installed from the site.
> 
> I also added notes to the readme for those wanting to run vagrant inside of a vmware-vm. My setup is similar.

I just tested your updates on a new vagrant box and it works so I'm merging.
